index.html

```diff
@@ -54,50 +55,46 @@
     </header>
     <section class="homepage" id="home">
         <div class="home">
-            <h3 class="hello">"Hello! Sharing world"</h3>
-            <h1 class="hello_title">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
-            <P class="hello_description">A joyfull celebration believing in the value of oppenness and sharing creating
-                a positive change with people
-                over 80 countries is taking place in October, in Pakistan.
-            </P>
+            <h3 class="hello">"Hello & Asalam-o-Alaikum! Trendy World"</h3>
+            <h1 class="hello_title">PAKISTAN WRIST WATCH CONVENTION 2022</h1>
+            <P class="hello_description">The brand exhibition distinguishes itself by offering a more intimate and engaging experience with world renowned horological brands. 
+                With over 40 luxury marques, the exhibition space allows visitors to immerse themselves in the unique spirits of each brand.</P>
+                <h2 class="hello_date">2022.01.17(MON) ~ 23(SUN)</h2>
+                <p class="hello_venue">@ Hotel ONE of Faisalabad, Pakistan </p>
```

Comment:
- [ ] Let's make these texts readable a bit. They currently strains the eye and that is not what we would want the user to experience. We can make it readable by adjusting the contrast a bit.
![image](https://user-images.githubusercontent.com/45668340/147161301-b399ef5d-dd4c-4867-afa0-049102228749.png)

Reply:
Let's do the same to this:point_down:
![image](https://user-images.githubusercontent.com/45668340/147161389-77b44eaf-bf98-4662-8cc6-81adc2db1659.png)